### ISSUE TITLE
distal ulna dorsal/volar displacement

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10748]
**Preferred term label:**
distal ulna dorsal/volar displacement

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lateral wrist X-ray showing volar or dorsal displacement of the distal ulna compared to the distal radius

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal radioulnar joint (DRUJ) instability
Triangular fibrocartilage complex (TFCC) tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I suspect there may be a difference between this and the existing term [Volar intercalated segment instability HP:6001209](https://hpo.jax.org/browse/term/HP:6001209), but they seem quite similar and it would be good to be precise about what the difference is (if any)?

**Comment by MickeySegal** [comment]
I've asked the orthopedist to answer this.

**Comment by MickeySegal** [comment]
The orthopedist replied:

Yes, those are two different findings. Volar intercalated segment instability (VISI) refers to a carpal bone instability pattern, whereas distal ulna volar/dorsal displacement refers to motion of the distal ulna due to an injury. Two separate bones and caused by separate pathologies, so they should be left separate.

**Comment by pnrobinson** [comment]
@MickeySegal thanks!
I think that one of the two terms overlaps with this term: [Dorsal subluxation of ulna HP:0006459](https://hpo.jax.org/browse/term/HP:0006459). Why "subluxation" and "displacement" are not synonyms, I do not know if clinically it is worth distinguishing between both.
Here is also an article where the term "Distal radioulnar joint dislocation" is used with "dorsal" and "ventral" -- I think this is probably also a synonym?
https://radiopaedia.org/articles/distal-radioulnar-joint-dislocation

**Comment by MickeySegal** [comment]
[Dorsal subluxation of ulna HP:0006459](https://hpo.jax.org/browse/term/HP:0006459) seems like a subset of the term we are requesting.

**Comment by pnrobinson** [comment]
@MickeySegal 
So I am wondering then if we should repurpose the existing term "[Dorsal subluxation of ulna HP:0006459](https://hpo.jax.org/browse/term/HP:0006459)" into a term called "distal ulna dorsal/volar displacement" -- this would make sense if there is no important clinical distrinction?

A possible definition could be
An abnormality of the distal radioulnar joint (DRUJ), which allows supination and pronation of the distal forearm and wrist. DRUJ injury and instability can can manifest with dorsal dislocation of the ulna (presenting with a dorsoulnar prominence and a block to supination) or with volar dislocation (presenting with a volar-ulnar prominence and block to pronation). Initial radiographic evaluation should start with a true posteroanterior (PA) and lateral of the affected forearm, wrist, and elbow. Lateral wrist radiographs show volar or dorsal displacement of the distal ulna compared to the distal radius.
PMID: [34539128](https://pubmed.ncbi.nlm.nih.gov/34539128/)

Comments:
The stability of the DRUJ can also be assessed by physical examination.


Alternatively, we could add a second term "Volar subluxation of ulna" if clinically it is important to distinguish?

**Comment by MickeySegal** [comment]
That seems reasonable, but as I went to add it I see that I had also tagged this finding with https://hpo.jax.org/browse/term/HP:5000049

I can't find the edit history on this.
